exercises/minesweeper/description.md

```diff
@@ -1,25 +1,35 @@
-Add the numbers to a minesweeper board.
+Add the score totals to a completed Minesweeper board.
 
 Minesweeper is a popular game where the user has to find the mines using
 numeric hints that indicate how many mines are directly adjacent
 (horizontally, vertically, diagonally) to a square.
 
 In this exercise you have to create some code that counts the number of
-mines adjacent to a square and transforms boards like this (where `*`
-indicates a mine):
-
-    +-----+
-    | * * |
-    |  *  |
-    |  *  |
-    |     |
-    +-----+
-
-into this:
-
-    +-----+
-    |1*3*1|
-    |13*31|
-    | 2*2 |
-    | 111 |
-    +-----+
+mines adjacent to a given empty square and replaces that square with the
+count.
+
+The board is a rectangle composed of blank space (' ') characters. A mine 
+is represented by an asterix ('\*') character.
```

Comment:
I usually see it spelled "asterisk", but I'm open to other spellings. But I couldn't find a resource online that told me that asterix was a valid spelling for the character, after I tried hard to do so. Open to being educated.

Reply:
No, you're correct ... apparently my computer knows the name of the French cartoon character and thought I was just being creative with case. I'll fix that.